client/components/formatted-header/index.tsx

```diff
@@ -1,26 +1,42 @@
 import clsx from 'clsx';
-import PropTypes from 'prop-types';
+import { ElementType, FC, PropsWithChildren, ReactNode } from 'react';
 import InfoPopover from 'calypso/components/info-popover';
 import { preventWidows } from 'calypso/lib/formatting';
-
 import './style.scss';
 
-function FormattedHeader( {
-	brandFont,
-	id,
+interface Props extends PropsWithChildren {
+	id?: string;
+	className?: string;
+	brandFont?: boolean;
+	headerText: ReactNode;
+	subHeaderAs?: ElementType;
+	subHeaderText?: ReactNode;
+	tooltipText?: ReactNode;
+	compactOnMobile?: boolean;
+	isSecondary?: boolean;
+	align?: 'center' | 'left' | 'right';
+	subHeaderAlign?: 'center';
+	hasScreenOptions?: boolean;
+	children?: ReactNode;
+	screenReader?: ReactNode;
+}
+
+const FormattedHeader: FC< Props > = ( {
+	brandFont = false,
+	id = '',
 	headerText,
 	subHeaderText,
 	tooltipText,
 	className,
-	compactOnMobile,
-	align,
+	compactOnMobile = false,
+	align = 'center',
 	subHeaderAlign,
-	isSecondary,
+	isSecondary = false,
 	hasScreenOptions,
 	subHeaderAs: SubHeaderAs = 'p',
 	children,
-	screenReader,
-} ) {
+	screenReader = null,
+} ) => {
```

Comment:
Thought: would it help to have these in alphabetical order? That would apply here and in `Props`.

Reply:
@daledupreez  fixed here 958d65078ebaac463e961e47231b02ef7f157c06